io/subsystem/src/main/java/org/wildfly/extension/io/OptionAttributeDefinition.java

```diff
@@ -137,7 +138,7 @@ private void setType() {
                 }
 
             } catch (Exception e) {
-                e.printStackTrace();
+                IOLogger.ROOT_LOGGER.error(e.getLocalizedMessage(), e);
```

Comment:
This should catch the specific checked exceptions the code can throw, wrap them in RuntimeException and rethrow. We shouldn't be swallowing problems here.

Reply:
@bstansberry I utterly agree with that, but here I just wanted to maintain the current behavior. If we know rethrow exception, its does mean that "things that work but printed stacktraces" are now going to "fails" (or at least throw exception). 

For Wildfly, as the next release is 11..0.0. that seems OK to do so, and I will update my PR right away, but I wonder if we should have the same strategy for EAP 7. (see https://github.com/jbossas/jboss-eap7/pull/2426)